app/helpers/users_helper.rb

```diff
@@ -160,36 +157,37 @@ def authors_header(collection, what = "People")
 
   def log_item_action_name(action)
     if action == ArchiveConfig.ACTION_ACTIVATE
-      t('users_helper.log_validated', :default => 'Account Validated')
+      t('users_helper.log_validated', default: 'Account Validated')
     elsif action == ArchiveConfig.ACTION_ADD_ROLE
-      t('users_helper.log_role_added', :default => 'Role Added: ')
+      t('users_helper.log_role_added', default: 'Role Added: ')
     elsif action == ArchiveConfig.ACTION_REMOVE_ROLE
-      t('users_helper.log_role_removed', :default => 'Role Removed: ')
+      t('users_helper.log_role_removed', default: 'Role Removed: ')
     elsif action == ArchiveConfig.ACTION_SUSPEND
-      t('users_helper.log_suspended', :default => 'Suspended until ')
+      t('users_helper.log_suspended', default: 'Suspended until ')
     elsif action == ArchiveConfig.ACTION_UNSUSPEND
-      t('users_helper.log_lift_suspension', :default => 'Suspension Lifted')
+      t('users_helper.log_lift_suspension', default: 'Suspension Lifted')
     elsif action == ArchiveConfig.ACTION_BAN
-      t('users_helper.log_ban', :default => 'Suspended Permanently')
+      t('users_helper.log_ban', default: 'Suspended Permanently')
     elsif action == ArchiveConfig.ACTION_WARN
-      t('users_helper.log_warn', :default => 'Warned')
+      t('users_helper.log_warn', default: 'Warned')
     elsif action == ArchiveConfig.ACTION_RENAME
-      t('users_helper.log_rename', :default => 'Username Changed')
-		elsif action == ArchiveConfig.ACTION_PASSWORD_RESET
-      t('users_helper.log_password_change', :default => 'Password Changed')
-		elsif action == ArchiveConfig.ACTION_NEW_EMAIL
-      t('users_helper.log_email_change', :default => 'Email Changed')
+      t('users_helper.log_rename', default: 'Username Changed')
+    elsif action == ArchiveConfig.ACTION_PASSWORD_RESET
+      t('users_helper.log_password_change', default: 'Password Changed')
+    elsif action == ArchiveConfig.ACTION_NEW_EMAIL
+      t('users_helper.log_email_change', default: 'Email Changed')
+    elsif action == ArchiveConfig.ACTION_TROUBLESHOT
+      t('users_helper.log_validated', default: 'Account Troubleshot')
```

Comment:
Even if we're not currently using it, this should be `log_troubleshot` or something -- right now it duplicates another key from line 160